package.json

```diff
@@ -46,7 +47,7 @@
     "express": "^4.14.0",
     "inflection": "^1.6.0",
     "loopback-connector-remote": "^1.0.3",
-    "loopback-datasource-juggler": "^3.0.0-alpha.1",
+    "loopback-datasource-juggler": "^3.0.0-alpha.6",
```

Comment:
Haha now it should be (there was no ^3.0.0-alpha.7 when I submitted this PR). ;)